services/kube-prometheus-stack/44.2.1/grafana-dashboards/controller-runtime.json

```diff
@@ -0,0 +1,710 @@
+{
+  "__inputs": [
+    {
+      "name": "DS_PROMETHEUS",
+      "label": "Prometheus",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus"
+    }
+  ],
+  "__requires": [
+    {
+      "type": "datasource",
+      "id": "prometheus",
+      "name": "Prometheus",
+      "version": "1.0.0"
+    }
```

Comment:
these differ from existing dashboards - https://github.com/mesosphere/kommander-applications/blob/cbuto/add-controller-runtime-dash/services/kube-prometheus-stack/44.2.1/grafana-dashboards/flux-cluster-stats.json
like there is no mention of `version: 1.0.0` and datasource is always referred to as `"datasource": "$datasource",`

I believe you might be just copy-pasting and dont know what needs to be done differently to get the right template though.

Reply:
ah yeah, let me drop those fields

Reply:
done in 4d94fd7d2048a1fbd6f85536410e779aa62493eb